pillarbox-player/src/main/java/ch/srgssr/pillarbox/player/PillarboxPlayer.kt

```diff
@@ -29,6 +30,10 @@ class PillarboxPlayer private constructor(private val exoPlayer: ExoPlayer) :
         addListener(this)
     }
 
+    fun printLog(message: String) {
```

Comment:
⚠️ **[detekt]** <detekt.UndocumentedPublicFunction> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The function printLog is missing documentation.